owasp-top10-2021-apps/a9/games-irados/app/routes.py

```diff
@@ -27,7 +27,8 @@
 bootstrap = Bootstrap(app)
 
 app.config.from_pyfile('config.py')
-
+logging.basicConfig(filename='log.log', level=logging.DEBUG, format=f'%(asctime)s %(levelname)s %(name)s %(threadName)s : %(message)s')
+#logging.basicConfig(level=logging.DEBUG, format=f'%(asctime)s %(levelname)s %(name)s %(threadName)s : %(message)s', handlers=[logging.FileHandler("ola_povo.log"),logging.StreamHandler()])
```

Comment:
Commenting code is a quick way to test different implementations, but it is not a good practice to maintain it after we finish the development. Could you remove the comment?
Why are we using `level=logging.DEBUG`? All the calls seems to be with `logger.info`. Using `DEBUG` could pollute the output (maybe some library will log debug messages?)

Reply:
Logs were wrongly configured. I fixed the settings in this new commit! thanks for the feedback!!

Reply:
Fixed!
<img width="995" alt="Captura de Tela 2022-05-02 às 12 53 40" src="https://user-images.githubusercontent.com/28883132/166265998-86a70da7-b69d-4634-b92f-809fb3516744.png">